### ISSUE TITLE
alt text missing for some GCconnex icons

### ISSUE THREAD

**Issue by Troy-Lawson** [tbs-sct/gcconnex#302]
GCCON-101
Some GCconnex icons are missing the Alt Text. If you hover over the icons, some of the icons have an alt tyext box that appears to explain what the ison does, but some icons don't have it. Ex: History (pages), Répondre (in French, doesn't work in the newsfeed - but works in the Wire feed). There might be others. 

 Thanks.
